controllers/helmchart_controller_test.go

```diff
@@ -694,6 +694,40 @@ var _ = Describe("HelmChartReconciler", func() {
 					storage.ArtifactExist(*got.Status.Artifact)
 			}, timeout, interval).Should(BeTrue())
 
+			When("Setting valid valuesFiles attribute", func() {
+				updated := &sourcev1.HelmChart{}
+				Expect(k8sClient.Get(context.Background(), key, updated)).To(Succeed())
+				chart.Spec.ValuesFiles = []string{
```

Comment:
```suggestion
				updated.Spec.ValuesFiles = []string{
```
Should this be `updated`? The originals are updating the chart defined outside of this function.